doc/auxiliary-programs/src/weather-converter-program/definitions-file-custom-file-processing.tex

```diff
@@ -212,35 +212,35 @@ \subsubsection{Field: DataElements}\label{field-dataelements}
 \midrule
 \endhead
 
-year & Year & - & n \tabularnewline
-month & Month & - & y \tabularnewline
-day & Day & - & y \tabularnewline
-hour & hour & - & y \tabularnewline
-minute & minute & - & n \tabularnewline
-datasource & datasource & - & n \tabularnewline
-drybulb & dry\_bulb\_temperature & C & y \tabularnewline
-dewpoint & dew\_point\_temperature & C & y \tabularnewline
-relhum & relative\_humidity & \% & y \tabularnewline
-atmos\_pressure & atmospheric\_pressure & Pa & y \tabularnewline
-exthorrad & extraterrestrial\_horizontal\_radiation & Wh/m2 & n \tabularnewline
-extdirrad & extraterrestrial\_direct\_normal\_radiation & Wh/m2 & n \tabularnewline
-horirsky & horizontal\_infrared\_radiation\_intensity\_from\_sky & Wh/m2 & y \tabularnewline
-glohorrad & global\_horizontal\_radiation & Wh/m2 & n \tabularnewline
-dirnorrad & direct\_normal\_radiation & Wh/m2 & y \tabularnewline
-difhorrad & diffuse\_horizontal\_radiation & Wh/m2 & y \tabularnewline
-glohorillum & global\_horizontal\_illuminance & lux & n \tabularnewline
-dirnorillum & direct\_normal\_illuminance & lux & n \tabularnewline
-difhorillum & diffuse\_horizontal\_illuminance & lux & n \tabularnewline
-zenlum & zenith\_luminance & lux & n \tabularnewline
-winddir & wind\_direction & degrees & y \tabularnewline
-windspd & wind\_speed & m/2 & y \tabularnewline
-totskycvr & total\_sky\_cover & tenths & n \tabularnewline
-opaqskycvr & opaque\_sky\_cover & tenths & n \tabularnewline
-visibility & visibility & km & n \tabularnewline
-ceiling\_hgt & ceiling\_height & m & n \tabularnewline
-presweathobs & present\_weather\_observation & - & y \tabularnewline
-presweathcodes & present\_weather\_codes & - & y \tabularnewline
-precip\_wtr & precipitable\_water & mm & n \tabularnewline
+year & Year & - & N \tabularnewline
+month & Month & - & Y \tabularnewline
+day & Day & - & Y \tabularnewline
+hour & hour & - & Y \tabularnewline
+minute & minute & - & N \tabularnewline
+datasource & datasource & - & N \tabularnewline
+drybulb & dry\_bulb\_temperature & C & Y \tabularnewline
+dewpoint & dew\_point\_temperature & C & Y \tabularnewline
+relhum & relative\_humidity & \% & Y \tabularnewline
+atmos\_pressure & atmospheric\_pressure & Pa & Y \tabularnewline
+exthorrad & extraterrestrial\_horizontal\linebreak\_radiation & $Wh/m^2$ & N \tabularnewline
```

Comment:
Lets typeset the units with the SI package (http://www.bakoma-tex.com/doc/latex/siunitx/siunitx.pdf). This one should be something like ```\si{\watt\hour\per\meter\squared}``` You might have to play around with it, but it should be something like that.

Reply:
I just changed it, could you check it please?

Reply:
FYI, there are a few documentation style tips in the [wiki](https://github.com/NREL/EnergyPlus/wiki/BuildingEnergyPlus). Search for Documentation and go [here](https://github.com/NREL/EnergyPlus/wiki/Documentation). Feel free to add more.

Reply:
@mjwitte Thanks! I have it bookmarked, will add to it if I come across something important.